### DISCUSSION TITLE
Can't edit an element. Error: "Properties may not have cycles"

### DISCUSSION THREAD

**Discussion by ValeMP** [Cotalker/documentation#750] [2. Questions and Answers]
Hola!

Estoy intentando editar una propiedad / elemento desde el Admin y no puedo guardar los cambios porque una de la subpropiedades tiene un ciclo. Tengo la duda si es que las propiedades seguirán con este concepto.

Si es que es así, por lo menos mis datos están modelados de la siguiente manera:

Entiendo que el ciclo debería deberse entre Ubicación Técnica, equipamiento y grupo de falla. 

Vale mencionar que 
* AF = Adicional Field 
* SP = Subpropiedad 

![Mantenimiento V Final-BASE DE DATOS drawio](https://user-images.githubusercontent.com/42095617/157881935-c7f540d2-25d5-4df6-b034-aded8220b5b1.png)

Este modelo es de mantenimiento correctivo, y se debe al reporte de problemas asociado al sector, y no un equipo en específico.

Ambiente: Staging
Propiedad: Puesto de Trabajo o Workstation
Producto: Mantenimiento correctivo

Aquí está la response de la llamada de red:

<img width="1414" alt="Captura de Pantalla 2022-03-11 a la(s) 10 40 50" src="https://user-images.githubusercontent.com/42095617/157880160-265468b3-fc79-4caf-b119-577332eff6a7.png">

**Comment by acedward** [comment]
Bug reportado y corregido.
